linkerd/telemeter/usage/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -163,10 +162,13 @@ class UsageDataTelemeter(
   log.info(s"connecting to usageData proxy at $metricsDst")
   val client = Client(metricsService, config, pid, orgId)
 
-  val adminHandlers = Seq(
-    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry))
+  override def adminHandlers: Seq[Handler] = Seq(
+    Handler("/admin/metrics/usage", new UsageDataHandler(metricsService, config, pid, orgId, registry)),
+    Handler("/usage", new UsageDataAdminHandler)
```

Comment:
I think to do this without adding the templates and js to the admin project, you'll need to add them to the usage project and also add handlers to serve them.